.travis/opensuse/travis-tasks.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+#Exit on failures
+set -e
+
+set -x
+
+JOB_NAME=${TRAVIS_JOB_NAME:Opensuse tumbleweed}
```

Comment:
This will take the JOB_NAME directly from the job title in .travis.yml. You will want to make sure that the parser you're using matches. Right now, you have 'tumbleweed' as the fallback default release but you use 'Tumbleweed' as the job name in .travis.yml.  Similarly, you use 'Opensuse' here and 'openSUSE' in the .travis.yml.

I'd prefer if you used the convention of the release portion being lowercase (so 'tumbleweed'). Since the openSUSE project prefers that capitalization, let's use that for the os_name.